modules/sdk-coin-eth/src/eth.ts

```diff
@@ -884,6 +899,52 @@ export class Eth extends BaseCoin {
     return response;
   }
 
+  /**
+   * Helper function for recover()
+   * This transforms the unsigned transaction information into a format the BitGo offline vault expects
+   * @param txInfo
+   * @param ethTx
+   * @param userKey
+   * @param backupKey
+   * @param gasPrice
+   * @param gasLimit
+   * @param eip1559
+   * @param replayProtectionOptions
+   * @returns {Promise<OfflineVaultTxInfo>}
+   */
+  async formatForOfflineVaultTSS(
+    txInfo: UnformattedTxInfo,
+    ethTx: EthTxLib.Transaction | EthTxLib.FeeMarketEIP1559Transaction,
+    userKey: string,
+    backupKey: string,
+    gasPrice: Buffer,
+    gasLimit: number,
+    backupKeyNonce: number,
+    eip1559?: EIP1559,
+    replayProtectionOptions?: ReplayProtectionOptions
+  ): Promise<OfflineVaultTxInfo> {
```

Comment:
I think this is just to keep the return type the same as is expected from the umbrella `recover` method, since the nonTSS recover path does use a promise.

Reply:
The umbrella will need to be updated then as well. We don't want to confuse our end users with bad types.